<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профиль</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icon.ico</normaloff>images/icon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="name_value_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="surname_label">
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="surname_value_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="patronymic_label">
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="patronymic_value_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="library_email_label">
       <property name="text">
        <string>Библиотека:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="library_email_value_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="ticket_contract_number_label">
       <property name="text">
        <string>Номер договора:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ticket_contract_number_value_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="pasport_phone_label">
       <property name="text">
        <string>Номер паспорта:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pasport_phone_value_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="SNILS_birthdate_label">
       <property name="text">
        <string>Номер снилс:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SNILS_birthdate_value_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="INN_label">
       <property name="text">
        <string>ИНН:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="INN_value_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
